### ISSUE TITLE
Shouldn't external be "video.js" and not "videojs" ?

### ISSUE THREAD

**Issue by gaspard** [collab-project/videojs-record#497]
https://github.com/collab-project/videojs-record/blob/a623a1bf147e0f41935a4cce33d17607407ebe52/build-config/fragments/common.js#L44

The current setting creates invalid "require" calls in compiled JS:

```js
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory(require("RecordRTC"), require("videojs"));
	else if(typeof define === 'function' && define.amd)
		define("VideojsRecord", ["RecordRTC", "videojs"], factory);
	else if(typeof exports === 'object')
		exports["VideojsRecord"] = factory(require("RecordRTC"), require("videojs"));
	else
		root["VideojsRecord"] = factory(root["RecordRTC"], root["videojs"]);
})
```

**Comment by thijstriemstra** [comment]
@gaspard sounds good, please open a pull request!

**Comment by thijstriemstra** [comment]
I opened a pull request for this, can you test #501 @gaspard?

**Comment by thijstriemstra** [comment]
Fixed in v4.1.0.
